index.js

```diff
@@ -193,7 +193,7 @@ class XiaomiRoborockVacuum {
 
     if (this.config.pause) {
       this.services.pause = new Service.Switch(
-        `${this.config.name} Pause`,
+        `${this.config.name} ${this.config.pauseWord}`,
```

Comment:
For backwards compatibility, can we default back to "Pause"?

```suggestion
        `${this.config.name} ${this.config.pauseWord || 'Pause'}`,
```

Reply:
Changed the default handling to the existing approach (see line 105)